chrome-extension/.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+.idea
+.vscode
```

Comment:
These are not project-related files, thus I depart from the assumption they should be part of a user's .`gitignore` not part of the project's `.gitignore` — for example, [check this global `.gitignore` tutorial](https://riptutorial.com/git/example/1222/a-global--gitignore-file).

Reply:
```suggestion
```